chore(news): add 11.0.1 release post (#1189)
PrismLauncher/prismlauncher.org: src/content/news/release-11.0.1.md

---
title: "Prism Launcher Release 11.0.1, now available"
description: "Symmetry not included!"
date: 2026-04-08
slug: "release-11.0.1"
release_version: "11.0.1"
minimum_macos_version: 12.0.0
macos_file_extension: zip
macos_signature: DHUm2PDTq3jJJk+Mq7mkwNQFjqFrC+LlCpJAhoOMEvCUyCS4ZZFOtz2C/EI2QTdwEcuuQxdmkDZvIfq70ZLDAQ==
tags:
  - Release
---

Prism Launcher 11.0.1 is Ready
Hello again everyone. As is usually the case, when a new release of a long running code base goes live the wider userbase can find more issues.

This is a patch release to fix a few fairly critical bugs with launching older Minecraft versions and fix Modpack updates. We've also restored the ability to view changelogs of Modrinth modpacks on the Instance's pack page.

> Run to [grab the latest download here](/download), and read on to see all the new stuff!

Reviewer: ```suggestion
Run to [grab the latest download here](/download), and read on to see the full list of changes.
```

## Changelog

> And for those who are a bit more curious (or just like reading long lists), here's our full changelog. You can also see the individual changes [here](https://github.com/PrismLauncher/PrismLauncher/compare/11.0.0...11.0.1).

Reviewer: ```suggestion
You can also see the individual changes [here](https://github.com/PrismLauncher/PrismLauncher/compare/11.0.0...11.0.1).
```
